### PR TITLE
Update notes to outline.md

### PR DESCRIPTION
**Pull request by callaghanmt**
Based on earlier Github discussion- expanded notes on content- file permissions, resource requests and task arrays as specific elements for delivery.

### CODE REVIEW

File: outline.md [hunk 1]
```diff
@@ -34,50 +34,48 @@ and how tasks get assigned and executed on the cluster.
...
+> Lola was hired by a research lab to help prepare the purchase of a multi-million dollar experiment.
 The experiment is known to fail at temperatures that are too low or too high.
-So Lola's task is to research the temperatures at a certain section of the lab room, where the device
-will be placed.
+So Lola's task is to research the temperatures in a certain section of the lab where the device will be placed.
 She knows that the temperature changes follow a daily pattern,
 and she's written some code to simulate these temperature changes.
 After running this code and generating the temperature predictions,
-she determine how closely her predictions match the actual temperature readings
+she determines how closely her predictions match the actual temperature readings
```
Comment (shwina) [inline outline.md:53]: "must determine" was what I meant to write here :-)

File: outline.md [hunk 2]
```diff
@@ -34,50 +34,48 @@ and how tasks get assigned and executed on the cluster.
 Learners will understand how nodes, CPUs, memory, etc., are organized,
 and "how it looks like one remote computer when it has a bazillion CPUs and several Terabytes?".
 At the end of the section, they will
-submit a batch job, trace its execution, and examine its results.
+submit a number of batch jobs for serial, parallel and task array workloads , trace their execution, and examine their results.
 Throughout, it will be stressed that different HPC sites
 have different setups, schedulers, policies and resources,
-and how learners can translate the concepts accordingly.
+and how learners can translate the concepts accordingly. The conepts we present will allow learners to quickly apply the techniques to their local clusters.
```
Comment (shwina) [inline outline.md:40]: conepts -> concepts

File: outline.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Accordingly, this section will begin by making learners feel
 comfortable in a shell environment by having them perform basic tasks
 like navigating the filesystem and manipulating its contents.
 But the latter half of the section will be focused on concepts
-like redirection, piping, and shell scripts,
+like redirection, piping, shell scripts and file permissions
```
Comment (shwina) [inline outline.md:27]: pedantic comment: comma after "file permissions" (?)

**Comment by shwina** [comment]
Thanks, @callaghanmt! Just some minor comments

**Comment by shwina** [comment]
Also, not sure if you meant to PR here or to swcarpentry/sc16-tutorial-proposal. I hope you'll be listed as a contributor in either case.

File: outline.md [hunk 4]
```diff
@@ -34,50 +34,48 @@ and how tasks get assigned and executed on the cluster.
...
 The simuation would take too long to run
 and generate too much data for her lab workstation,
 so she will use the local University's HPC facility for this work.
 
-They will then walk through the different steps involved, including
-data transfer,
-software installation and
-developing batch scripts to submit jobs for
-performing the simulation and analyzing the results.
+They will then guide learners through the different steps involved, including
```
Comment (shwina) [inline outline.md:59]: "They" here refers to "the learners" - so maybe "they will then be guided through"? Although I think this makes it sound a bit passive on the learners' part.

**Comment by callaghanmt** [comment]
Here, probably! I'm going to add in a PR to swcarpentry/sc16-tutorial-proposal about providing additional HPC resource here at Leeds on one of our clusters to demonstrate SGE alongside PBS.

File: outline.md (hunk 4 above)
Comment (callaghanmt) [inline outline.md:59]: I get what you mean now- I think any way works.

**Comment by shwina** [comment]
Awesome!

**Pull request merged**